manual/manual_circuit.rst

```diff
@@ -1652,14 +1703,16 @@ To add gates or boxes to a circuit with specified op group names, simply pass th
     # Substitute a new 1-qubit circuit for all ops in the "rotations" group:
     newcirc = Circuit(1).Rx(0.125, 0).Ry(0.875, 0)
     circ.substitute_named(newcirc, "rotations")
+
     # Replace the "special one" with a different op:
     newop = Op.create(OpType.T)
     circ.substitute_named(newop, "special one")
+
     # Substitute a box for the "Fred" group:
-    newcbox = CircBox(Circuit(2).H(1).CX(1,0))
+    newcbox = CircBox(Circuit(2, name="C'").H(1).CX(1, 0))
```

Comment:
C' is probably not as clear to read in this context as a different letter

Reply:
I've opted for `P` and `Q` instead. Hopefully this is clearer for the reader.